GUI Rework

markjfine opened the pull request:
Added:
1. Buttons to select streams
2. Resizable album art
3. Additional info on the info tab (e.g., Message, Stream/Stream Descriptions, Data Services, Alert)
4. Genre on Track Info
5. Newly found ServiceDataType and MIMEType
6. Callsign + Slogan to Frequency line
7. Synch/Lost Device pilot to status bar

markjfine committed:
Add files via upload

Added:
1. Buttons to select streams
2. Resizable album art
3. Additional info on the info tab (e.g., Message, Stream/Stream Descriptions, Data Services, Alert)
4. Genre on Track Info

markjfine committed:
Add files via upload

markjfine committed:
Add files via upload

markjfine committed:
Add files via upload

1. Reordered the Info tab.
2. Fixed the way the Programs and Data Services were listed to be more relevant (and stable)
3. Made the Program Buttons ellipcized to prevent expanding the window
4. Removed the Slogan from the Frequency line to prevent expanding the window

andrewfer000 reviewed (approved):
Everything looks good. I approve it. Now we just need the onwer's approval.


Oh one more suggestion - what you did for the album art, do for the maps tab if you can.

markjfine committed:
Revised traffic/weather map tab display

1. Made images fill available space in the tab
2. Images resize when window resizes

markjfine committed:
Program Type / Error Status

1. Completed source of stream data (program type) in the Info tab to SIG from SIS messages in order to reduce number of label writes.
2. Changed BER number in status bar from Average to Current to make it a more useful tuning tool.
